src/main/java/org/opensearch/sdk/handlers/ClusterSettingsResponseHandler.java

```diff
@@ -13,16 +13,14 @@
 import org.apache.logging.log4j.Logger;
 import org.opensearch.cluster.ClusterSettingsResponse;
 import org.opensearch.common.io.stream.StreamInput;
-import org.opensearch.sdk.ExtensionsRunner;
 import org.opensearch.threadpool.ThreadPool;
 import org.opensearch.transport.TransportException;
 import org.opensearch.transport.TransportResponseHandler;
-import org.opensearch.transport.TransportService;
 
 import java.io.IOException;
 
 /**
- * This class handles the response from OpenSearch to a {@link ExtensionsRunner#sendClusterSettingsRequest(TransportService)} call.
+ * This class handles the response from OpenSearch to a {@link org.opensearch.sdk.SDKTransportService#sendClusterSettingsRequest()} call.
```

Comment:
This is fine, but I think it would be more readable if you made `SDKTransportService` an import instead of putting the fully qualified class name here (and a few other places in this PR)